### PR TITLE
Standardize the order of imports packages

### PR DESCRIPTION
**Pull request by wgliang**
In accordance with the standard packages, internal packages, external packages order import all packages.

### CODE REVIEW

**Comment by wgliang** [comment]
Who will review this code? Please give feedback.

**Comment by xiaozhu36** [comment]
HI @wgliang I am so sorry for giving a late feedback. And thank you very much for your contribution. Now, there are too many conflicts about the PR and I can not resolved them by Web browser. Could you please submit the PR again after resolving these conflicts? After that I will review them right now. Thanks a lot.

Regards
He Guimin

**Comment by wgliang** [comment]
I have resolved the conflict, you can not merge this PR If you do not think it is important, but since the open source project as much as possible to regulate the code, unit testing, etc. I think a good project code must be standardized.

GOOD LUCK!

**Comment by wgliang** [comment]
I have resolved the conflict, you can not merge this PR If you do not think it is important, but since the open source project as much as possible to regulate the code, unit testing, etc. I think a good project code must be standardized.

GOOD LUCK!

**Comment by wgliang** [comment]
Have DONE It!!

File: alicloud/resource_alicloud_slb.go [hunk 1]
```diff
@@ -1,12 +1,17 @@
...
 import (
+	"bytes"
+	"errors"
 	"fmt"
+	"reflect"
+	"strings"
+	"time"
 
+	"github.com/denverdino/aliyungo/common"
 	"github.com/denverdino/aliyungo/slb"
```
Comment (xiaozhu36) [inline alicloud/resource_alicloud_slb.go]: I think line 4, 5, 7, 8 and 11 should be removed. These are useless code in the latest master branch.

File: alicloud/resource_alicloud_instance.go [hunk 2]
```diff
@@ -1,15 +1,17 @@
 package alicloud
 
 import (
+	"encoding/base64"
+	"encoding/json"
```
Comment (xiaozhu36) [inline alicloud/resource_alicloud_instance.go]: Please remove useless code of line 4 and line 5.

File: alicloud/resource_alicloud_ess_scalinggroup.go [hunk 3]
```diff
@@ -2,6 +2,10 @@ package alicloud
 
 import (
 	"fmt"
+	"strings"
+	"time"
+
+	"github.com/denverdino/aliyungo/common"
```
Comment (xiaozhu36) [inline alicloud/resource_alicloud_ess_scalinggroup.go]: Please remove useless codes of line 4-8.

**Comment by wgliang** [comment]
Strongly recommended to intervene in the CI platform, installation dependent took a long time. . . .

File: alicloud/resource_alicloud_instance.go (hunk 2 above)
Comment (wgliang) [reply alicloud/resource_alicloud_instance.go]: Already updated and I have been compiled locally.

**Comment by bittopaz** [comment]
This is a good one 👍

**Comment by xiaozhu36** [comment]
HI @wgliang Thanks for your suggestion, and I will consider it.

**Pull request merged**